Create Colour v0.3.9 version.

KelSolaar opened the pull request.

KelSolaar committed:
Create Colour v0.3.9 version.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

KelSolaar committed:
Update "SHA256" value.

Late update of the "PKG-INFO" and "README.rst" files resulted in modified "SHA256" value.

thomasmansencal commented:
@ocefpaf: I noticed that my build failed because of `curl` missing on the AppVeyor building machine, is there a way to restart the build without doing a new commit?

```
[00:00:00] Build started
[00:00:01] git clone -q https://github.com/conda-forge/colour-science-feedstock.git C:\projects\colour-science-feedstock
[00:00:04] git fetch -q origin +refs/pull/1/merge:
[00:00:05] git checkout -qf FETCH_HEAD
[00:00:17] Running Install scripts
[00:00:17] curl https://raw.githubusercontent.com/conda-forge/conda-forge-build-setup-feedstock/master/recipe/ff_ci_pr_build.py > ff_ci_pr_build.py
[00:00:17] 'curl' is not recognized as an internal or external command,
[00:00:17] operable program or batch file.
[00:00:17] Command exited with code 1
```

Cheers,

Thomas

thomasmansencal commented:
@ocefpaf: Scratch the above, I created an account and managed to link it!

Cheers,

Thomas

ocefpaf commented:
The lack of `curl` was a transient issue that is now fixed. Feel free to merge this. LGTM
